__snapshots__/index.test.js.snap

```diff
@@ -108,6 +110,8 @@ exports[`https://github.com/fregante/shorten-repo-url/tree/v0.12/doc 1`] = `<cod
 
 exports[`https://github.com/fregante/shorten-repo-url/wiki/%22Can-you-add-this-feature%3F%22 1`] = `Wiki: "Can you add this feature?"`;
 
+exports[`https://github.com/fregante/shorten-repo-url/wiki/한글-위키-페이지-경로 1`] = `Wiki: 한글 위키 페이지 경로`;
```

Comment:
If you have it, it would be good to see real URLs, especially because GitHub might vary how it handles these over time.

Also can you add a test for `#hash-on-the-wiki-page` like

```
https://github.com/refined-github/refined-github/wiki/Contributing#create-a-new-feature
```

Reply:
sure, would it be ok for me to use URL from my repo? (since i have no permission to edit wiki pages in refined-github)

Reply:
Yes sure

Reply:
done in 7302dcbd5f2d92d4d6b738d0af68e162b1927408